client/src/Movies/Movie.js

```diff
@@ -38,31 +39,20 @@ export default class Movie extends Component {
   // }
 
   render() {
-    if (!this.state.movie) {
-      return <div>Loading movie information...</div>;
+    const movie = this.state.movie; 
+      if (!movie) {
+        return <div>Loading movie information...</div>;
     }
 
-    const { title, director, metascore, stars } = this.state.movie;
     return (
-      <div className="save-wrapper">
-        <div className="movie-card">
-          <h2>{title}</h2>
-          <div className="movie-director">
-            Director: <em>{director}</em>
-          </div>
-          <div className="movie-metascore">
-            Metascore: <strong>{metascore}</strong>
-          </div>
-          <h3>Actors</h3>
-
-          {stars.map(star => (
-            <div key={star} className="movie-star">
-              {star}
-            </div>
-          ))}
-        </div>
-        <div className="save-button">Save</div>
-      </div>
+      <MovieCard
+        key = {movie.id}
+        movie = {movie}
+        title = {movie.title}
+        director = {movie.director}
+        metascore = {movie.metascore}
+        stars = {movie.stars} 
```

Comment:
No need to pass down the individual parts of the `movie` object if you're sending the entire object down as a prop.